src/commands/setup.rs

```diff
@@ -134,6 +134,70 @@ impl Setup {
+                    //Configure Bridge and veth_pairs
+                    let status_block = network::core::Core::macvlan_per_podman_network(
+                        per_network_opts,
+                        network,
+                        &self.network_namespace_path,
+                    )?;
+                    response.insert(net_name.to_owned(), status_block);
+
+                    let id_network_hash = network::core_utils::CoreUtils::create_network_hash(
+                        net_name,
+                        MAX_HASH_SIZE,
+                    );
+
+                    firewall_driver.setup_network(network.clone(), id_network_hash.clone())?;
+                    let port_bindings = network_options.port_mappings.clone();
+                    match port_bindings {
+                        None => {}
+                        Some(i) => {
+                            // Need to enable sysctl localnet so that traffic can pass
+                            // through localhost to containers
+                            let network_interface = &network.network_interface;
+                            match network_interface {
+                                None => {}
+                                Some(i) => {
+                                    let localnet_path =
+                                        format!("net.ipv4.conf.{}.route_localnet", i);
+                                    let sysctl_localnet = get_sysctl_value(localnet_path.as_str())?;
+                                    if sysctl_localnet != *"1" {
+                                        set_sysctl_value(localnet_path.as_str(), "1")?;
+                                    }
+                                }
+                            }
+                            let container_ips =
+                                &per_network_opts.static_ips.as_ref().ok_or_else(|| {
+                                    std::io::Error::new(
+                                        std::io::ErrorKind::Other,
+                                        "no container ip provided",
+                                    )
+                                })?;
+                            let networks = &network.subnets.as_ref().ok_or_else(|| {
+                                std::io::Error::new(
+                                    std::io::ErrorKind::Other,
+                                    "no network address provided",
+                                )
+                            })?;
+                            firewall_driver.setup_port_forward(
+                                &network_options.container_id,
+                                i,
+                                container_ips[0],
+                                networks[0].subnet,
+                                net_name,
+                                &id_network_hash.as_str()[0..MAX_HASH_SIZE],
+                            )?;
+                        }
+                    }
```

Comment:
macvlan does not need firewall setuop, please remove this

Reply:
removed in latest commit